guidelines.md

```diff
@@ -82,9 +82,10 @@ Some errors may also contain additional information relevant to the error on top
 ## Authentication
 
 Each Mews environment (e.g. demo, production) requires a different set of tokens. Please use the tokens below to connect to the Mews Demo environments. To sign into the system, use the following credentials:
-* **PlatformAddress** - https://demo.mews.li
+* **PlatformAddress** - `https://api.mews-demo.com`
```

Comment:
What about naming it `ApiBaseUrl` and `WebSocketBaseUrl`? BTW this file is really long now, we discussed with @jirrick and @patrikmiksovsky95  that it would be good to do quick polishing of the whole docs, not much work, just simple reorganization would make it much better and easier for new developers to use it. @jirrick do we have notes about it somewhere? Will we do it at some point?

Reply:
https://mews.myjetbrains.com/youtrack/issue/RND-45785